models/passenger.js

```diff
@@ -0,0 +1,110 @@
+const pg = require('pg-promise')()
+
+const connection =  {
+  host: 'localhost',
+  port: 5432,
+  database: 'lg-metrorail-db',
+  user: 'jhallman5',
```

Comment:
once I commented out this line the tests that pass went from 14 to 24. I dont see anywhere in your schema or database where you refer to users, I would square this up before trying to be more specific with you db connection.